internal/driver/gomobile/driver.go

```diff
@@ -23,7 +23,11 @@ import (
 	"fyne.io/fyne/theme"
 )
 
-const tapSecondaryDelay = 300 * time.Millisecond
+const (
+	tapMoveThreshold  = 4.0                    // how far can we move before it is a drag
```

Comment:
Is this and tapYOffset adjusted by scale? 4 pixels on a low end screen is much larger than 4 pixels on a high end screen

Reply:
Yes, this is a point value - it will scale per-screen. On my iPhone it is about 15 pixels.
I tweaked it until I was reliably able to tap a button when my finger felt like it was in the right place.

Reply:
Perfect